misc/header.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
```

Comment:
We can remove the entire misc directory. We still need to update the cookiecutter template: https://github.com/inveniosoftware/cookiecutter-invenio-module/issues/29